config.json

```diff
@@ -1,7 +1,7 @@
 {
   "definitions": {
     "npm": {
-      "default": "8.4.0",
+      "default": "8.10.0",
```

Comment:
Forgive my drive-by comment here, but I thought npm wasn't supported with corepack?

Reply:
It has always been supported just not enabled by default when you run `corepack enable`, you can explicitly enable it with `corepack enable npm`
Ref https://github.com/nodejs/corepack/pull/24

> Note that the npm shims will not be installed unless explicitly requested, as npm is currently distributed with Node.js through other means.
https://github.com/nodejs/corepack/blob/648bd95e947d0a6be0beba4c594de127d94da7d9/README.md#corepack-enable--name